composer.json

```
@@ -27,7 +27,7 @@
         "dg/bypass-finals": "*"
     },
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.4 || ^8.1",
```

[o] It's better to not have a ceiling on PHP version as it's generally well backwards-compatible. The upper bound will only make it harder to try out new versions and fix the support, eventually
```suggestion
        "php": "^7.4 || >=8.1",
```